src/utils/daos_hdlr.c

```diff
@@ -210,12 +210,10 @@ pool_get_prop_hdlr(struct cmd_args_s *ap)
 	assert(ap != NULL);
 	assert(ap->p_op == POOL_GET_PROP);
 
-	rc = daos_pool_connect(ap->p_uuid, ap->sysname,
-			       DAOS_PC_RO, &ap->pool,
-			       NULL /* info */, NULL /* ev */);
+	rc = daos_pool_connect(ap->p_uuid, ap->sysname, DAOS_PC_RO, &ap->pool, NULL, NULL);
 	if (rc != 0) {
-		fprintf(ap->errstream, "failed to connect to pool "DF_UUIDF
-			": %s (%d)\n", DP_UUID(ap->p_uuid), d_errdesc(rc), rc);
+		fprintf(ap->errstream, "failed to connect to pool "DF_UUIDF": %s (%d)\n",
+			DP_UUID(ap->p_uuid), d_errdesc(rc), rc);
```

Comment:
Was this one not meant to use the label / %s instead?

Reply:
yea, but i think this is just dead code since it's only used from daos_old tool.. anyway i will update

Reply:
actually this is a bigger change than i realized as there are many usage of this throughout, mostly with the old tool. iI will check those in a different PR. ill push some of the other fixes now here though.

Reply:
Fair enough. FWIW, this file has lots of inconsistencies with print statements :)

Reply:
indeed, and it was not my intention to fix all of it. i tried at least to fix the container uuid printing when using the daos tool. the pool ones seem mostly used from the daos_old tool since the pool handlers are implemented in the go side, and should be removed all together once we remove the daos.c (daos_old) tool

Reply:
DH_PERROR_SYS() and DH_PERROR_DER() are intended to resolve the inconsistencies in this file, but it's a lot of effort to update them all and as stated some of it appears dead code.  As long as we're moving in the right direction then I think that's good enough.  Once it's stabilized and new features have landed/code is removed then we can do a pass just to migrate things over but right now unless the output is changing anyway it's just adding noise/conflicts.

Reply:
Sure, makes sense!